### DISCUSSION TITLE
Am unable to use the AI Chatbot

### DISCUSSION THREAD

**Discussion by MilesErasmus** [ahmadfaizalbh/Chatbot#83] [General]
I keep getting a lot of error messages. Am I doing something wrong.
I am new to programming and has a basic python skill. 
I need you help

**Comment by MilesErasmus** [comment]
Here is one of the error message . 
Kindly help me to understand.

C:\Users\pc\AppData\Local\JetBrains\PyCharmCE2023.2\demo\PyCharmLearningProject\venv\Scripts\python.exe "C:/Program Files/JetBrains/PyCharm Community Edition 2023.2/plugins/python-ce/helpers/pydev/pydevd.py" --multiprocess --qt-support=auto --client 127.0.0.1 --port 50484 --file "C:\Users\pc\Desktop\PYTHON\AI CHABOT\Chatbot-master\setup.py" 
Connected to pydev debugger (build 232.8660.197)
C:\Users\pc\AppData\Local\JetBrains\PyCharmCE2023.2\demo\PyCharmLearningProject\venv\Lib\site-packages\setuptools\dist.py:771: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead
  warnings.warn(
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied

Process finished with exit code 1

**Reply by ahmadfaizalbh** [reply]
You are trying to execute `setup.py` without command 

```bash
C:\Users\pc\AppData\Local\JetBrains\PyCharmCE2023.2\demo\PyCharmLearningProject\venv\Scripts\python.exe "C:/Program Files/JetBrains/PyCharm Community Edition 2023.2/plugins/python-ce/helpers/pydev/pydevd.py" --multiprocess --qt-support=auto --client 127.0.0.1 --port 50484 --file "C:\Users\pc\Desktop\PYTHON\AI CHABOT\Chatbot-master\setup.py"
```

To install you have to use `python setup.py install`

**Reply by alpha3366666** [reply]
no found
